.github/workflows/tests.yml

```diff
@@ -97,34 +99,42 @@ jobs:
               --cov-append \
               --cov-report xml \
               -m "not serial" tests \
-              ${{ env.pytest-replay }}
+              ${{ env.pytest-replay }} \
+              --alluredir=allure-results
+          tar -zcf allure-results.tar.gz allure-results
 
       - name: Upload Pytest Replay
         uses: actions/upload-artifact@v2
         with:
           name: Linux_replay-py-${{ matrix.python-version }}_${{ matrix.conda-version }}_${{ matrix.test-type }}
-          path: pytest-replay/
+          path: pytest-replay/ 
```

Comment:
Configure your editor to remove trailing whitespace

Reply:
ok..